### ISSUE TITLE
How can I get the labels of the segment mask?

### ISSUE THREAD

**Issue by shamoons** [qubvel/segmentation_models#91]
```
from keras.preprocessing import image
from segmentation_models import Unet
import numpy as np
import imageio

model = Unet(backbone_name='resnet152',
             encoder_weights='imagenet', freeze_encoder=True)
model.compile('Adam', 'binary_crossentropy', ['binary_accuracy'])

img = image.load_img('seg_images/image1.png')
x = image.img_to_array(img)
x = np.expand_dims(x, axis=0)

mask = model.predict(x)
mask = mask.astype('uint8')
imageio.imwrite("result.png", mask[0])

```

First, the mask is all black. But assuming it works, how can I get the various labels for each segment?

**Comment by qubvel** [comment]
Hi @shamoons 
If I understand you correctly:
Round mask - `mask.round()`
 0 - is background
 1 - is target class

**Comment by shamoons** [comment]
Thanks for the quick reply @qubvel . What about for multiple classes in an image? And do I have to train it or can I use a pretrained model?

**Comment by qubvel** [comment]
This repo does not contain pretrained segmentation models (only encoders are pretrained), so you have to train network for any task.
For multiclass output is a 4D tensor of shape (batch, h, w, n_classes), so in each channel you will have a class.

**Comment by shamoons** [comment]
Are there any compatible pretrained segmentation models for cityscape?

**Comment by qubvel** [comment]
What do you mean compatible? Maybe someone has already trained one of the models, but I dont have such weights.

**Comment by shamoons** [comment]
Thanks

**Comment by sudo-install-MW** [comment]
https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/cityscapes.md
pretrained mobilenet v2 and xception, hope it helps

**Comment by shamoons** [comment]
How would I load it?
